CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
 ## 2.8.4 - 2022-11-11
-- [Bugfix]: fix: close virtualaccount end point [#310]
-- [Bugfix]: fix: passing `partial_payment` boolean issue when creating order [#311]
--  Added Issue and feature templates
+### Patch Changes
+ - Close virtualaccount end point  \[[`3e6daf3`](https://github.com/razorpay/razorpay-node/commit/3e6daf3c555f62eb23660a54eaae756e395ea3b6)]:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** 

- Fixed `virtualAccount.close` function implementation


- Fixed `orders.create` function by passing `partial_payment` boolean

Reply:
made this changes

Reply:
Lets start tagging who made these changes. Like how FE-universe or Blade does.

Reply:
[#310](https://github.com/razorpay/razorpay-node/pull/310) [`3e6daf3`](https://github.com/razorpay/razorpay-node/commit/3e6daf3c555f62eb23660a54eaae756e395ea3b6) : Thanks [@ankitdas13 ](https://github.com/snitin315)! - Fixed `virtualAccount.close` function implementation

[#311](https://github.com/razorpay/razorpay-node/pull/322) [`653a278`](https://github.com/razorpay/razorpay-node/commit/653a2784b57095170ee3c0b3dff877aa7d8afca4):  Thanks [@ankitdas13 ](https://github.com/snitin315)! - Fixed `orders.create` function by passing `partial_payment` boolean

@kkyusufk @sidag95 is it looking good now ?

Reply:
@ankitdas13 Sure, please update the changelog with the above, will approve

Reply:
@sidag95 It's done. Can you please approve?